dev/yarn.ts

```diff
@@ -91,6 +91,30 @@ const packageList: Fig.Generator = {
   },
 };
 
+// generate workspace argument completion
+const scriptList: Fig.Generator = {
+  script: function (context) {
+    return `cat ${context[context.length - 2]}/package.json`;
+  },
+  postProcess: function (out) {
+    if (out.trim() == "") {
+      return [];
+    }
+    try {
+      const packageContent = JSON.parse(out);
+      const scripts = packageContent["scripts"];
+      if (scripts) {
+        const scriptKeys = [];
+        for (let i = 0; i < Object.keys(scripts).length; i++) {
+          scriptKeys.push({ name: Object.keys(scripts)[i] });
+        }
+        return scriptKeys;
```

Comment:
We can improve this with a `map`:
```suggestion
        return Object.keys(scripts).map(script => ({ name: script }))
```